Makefile

```diff
@@ -147,4 +147,4 @@ clean: ## Clean dist files and delete all builds in electron/release
 	rm $(ELECTRON_DIR)/release/*
 
 help:
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Whitespaces should not pollute the PR diff. Make sure this file does not get listed .

Reply:
If you are using sublime text, add  `"ensure_newline_at_eof_on_save": true` and `"trim_trailing_white_space_on_save": true` to your preferences